src/Raven.Server/Documents/Handlers/Processors/AdminForbiddenUnusedIdsHandlerProcessorForGetUnusedIds.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Elastic.Clients.Elasticsearch;
+using JetBrains.Annotations;
+using Raven.Client.Documents.Operations;
+using Raven.Client.ServerWide;
+using Raven.Server.ServerWide.Context;
+
+namespace Raven.Server.Documents.Handlers.Processors;
```

Comment:
why the name of this file starts with `Admin..`

Reply:
also seems that the file name doesn't match here.

Reply:
Done